htdocs/install/mysql/tables/llx_product_association.sql

```diff
@@ -23,7 +23,6 @@ create table llx_product_association
   fk_product_pere       integer NOT NULL DEFAULT 0, -- id du produit maitre
   fk_product_fils       integer NOT NULL DEFAULT 0, -- id du sous-produit
   qty                   double NULL,
-  incdec                integer DEFAULT 1, -- when set to 1 changing stock of product will change stock of linked product too
```

Comment:
Why removing this. It breaks a future to allow user to decide if a line of a kit must have stock impacted if not (default is 1 but some use case need 0 for example when it contains a produt we don't want to follow stock)

Reply:
According to me,
in a virtual product the parent is never incremented or decremented, it's only on children you can increment or decrement. You increment only children on reception and you sell a virtual product that contain some others products and you decrement only children and not the virtual product.

Reply:
> According to me, in a virtual product the parent is never incremented or decremented, it's only on children you can increment or decrement. You increment only children on reception and you sell a virtual product that contain some others products and you decrement only children and not the virtual product.

Yes, And this field incdec is necessary to know if we increment or not the children products. Currently we can't set the property "must be stock managed" at product level. So we can use this to have virtual products that contains both children product that need to be increment or decremented and other that won't change (because we don't want to follow consumption for this product).